src/_gettsim/transfers/arbeitsl_geld_2/kindergelduebertrag.py

```diff
@@ -137,11 +143,11 @@ def _diff_kindergeld_kindbedarf_m(
     # the full amount for the child and one time the Kindergeldübertrag for the parent -
     # because the child doesn't drop out of Bedarfsgemeinschaft endogenously).
```

Comment:
But this is happening now, right? Or will that be part of a different PR?

Reply:
That will be part of a different PR. The `eigenbedarf_gedeckt` condition makes sure that parents and children are not in the same BG, but we can throw `eigenbedarf_gedeckt` out and adjust the function such that we don't need it. Will make an issue.